lib/feedback.js

```diff
@@ -246,6 +270,16 @@ function feedback(system) {
 		}
 	})
 
+	system.on('feedback_check_all_banks', function () {
```

Comment:
I think you can use `feedback_check_all` instead of this. No second parameter necessary

Reply:
Thought of it, but there's no need for this case to cycle through any triggers.  Your call.

Reply:
Oh good point, I didnt realise that was subscribed to over there also..
Perhaps just pull out the function here to a shared implementation then

Reply:
Yeah, I can do that.  `scheduler` listens to `feedback_check_all`, which I'm not keen about but it works.  Yeah I can abstract the interior calls for this and `feedback_check_all` because of that though.